### PR TITLE
Updated Molecules documentation.

### PR DESCRIPTION
**Pull request by mattiasberlamont**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/31-molecules/table-of-contents/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+* The web page does not contain that many text. In this case, the table of contents component is unecessary.
+* The amount of content on the web page will make the page too long or too slow to load. In this case, split the content accross multiple pages.
+* The content of the page is not clearly separated by headings. In this case, the table of contents component should not be used.
+
+## How it works
+
+### General
+
+The table of contents component is a list of anchor links. Each anchor link points to a section withing the web page. The sections typically have a clear title (typically h2 headings). The titles of the sections are used as the link text for the anchor links.
```
Comment (hstandaert) [inline styleguide/components/31-molecules/table-of-contents/README.md]:
```suggestion
The table of contents component is a list of anchor links. Each anchor link points to a section within the web page. The sections typically have a clear title (typically h2 headings). The titles of the sections are used as the link text for the anchor links.
```

File: styleguide/components/31-molecules/tabs/README.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+The tabs component allows users to switch between two or more related sections of content on the same web page, displaying one section at a time.
+
+## When to use this component
+
+Use the tabs component when:
+
+* Two or more related sections of content exist on a web page that can be clearly labelled.
+* Users typically are not interested to see all sections.
+* Users typically will not need to see all sections at once.
+* The first section is more relevant than the others sections for most users.
```
Comment (hstandaert) [inline styleguide/components/31-molecules/tabs/README.md]:
```suggestion
* The first section is more relevant than the other sections for most users.
```

File: styleguide/components/31-molecules/tabs/README.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+
+## When not to use this component
+
+Do not use a tabs component:
+
+* To hide content that is essential to all users.
+* When users will need to see all sections at once, for instance to compare information in the different sections.
+* When the sections are equally relevant for most users or when users typically are interested in a few specific sections. In this case, use an <a href="{{path './accordion.html'}}">accordion</a> instead.
+* When users have to read through all the related sections of content in order, for example, to understand a process.
+* To show progress or a linear flow. In these cases, use a multi-step form or a wizard instead.
```
Comment (hstandaert) [inline styleguide/components/31-molecules/tabs/README.md]:
```suggestion
<!-- @TODO add a link to the proposed components (after development is done) -->
* To show progress or a linear flow. In these cases, use a multi-step form or a wizard instead.
```
_FYI: This comment will not be displayed; it's just a reminder_

File: styleguide/components/31-molecules/teaser/README.md [hunk 4]
```diff
@@ -1,17 +1,37 @@
...
-## Functionality
+## When to use this component
 
-- The entire teaser block is clickable.
-- Links and buttons inside the teaser are clickable.
+Use the teaser component in a <a href="{{path './collection.html'}}">collection</a> to show teasers of content on overview pages.
+
+## When not to use this component
+
+Do not use the teaser component for other uses than in a <a href="{{path './collection.html'}}">collection</a> on overview pages.
```
Comment (hstandaert) [inline styleguide/components/31-molecules/teaser/README.md]: A teaser could be used at the bottom of some other content pages as well, such as related articles. That implies that you **can** use it on other pages besides an overview page.

File: styleguide/components/31-molecules/teaser/README.md [hunk 5]
```diff
@@ -1,17 +1,37 @@
...
+The entire teaser block is clickable. When clicking the teaser block, the primary teaser link is followed.
+
+The secondary teaser links inside the teaser block are also clickable. When clicking a secondary teaser link, that links is followed. When not clicking on a secondary teaser link but anywhere else in the teaser block, the primary teaser link is followed.
 
 ## Accessibility
 
-- Do not repeat links, links inside one molecule must have unique destinations.
-- The teaser overlay-link is excluded from the tabindex and invisible to
+* Do not repeat links, links inside one molecule must have unique destinations.
+* he teaser overlay-link is excluded from the tabindex and invisible to
```
Comment (hstandaert) [inline styleguide/components/31-molecules/teaser/README.md]:
```suggestion
* The teaser overlay-link is excluded from the tabindex and invisible to
```

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/teaser/README.md (hunk 4 above)
Comment (mattiasberlamont) [reply styleguide/components/31-molecules/teaser/README.md]: Correct, I updated the Teaser documentation to better reflect this.

**Comment by mattiasberlamont** [comment]
@hstandaert Thanks for reviewing. I committed your suggestions and updated the Teaser documentation.

**Review by hstandaert — APPROVED** [review]

File: styleguide/components/31-molecules/gent-info/README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Gentinfo
+
+Gentinfo is the central point of contact for information about the city council and the city of Ghent.
+
+## When to use this component
+
+Use the Gentinfo component to show Gentinfo contact information as a call-to-action on a web page when it is relevant to the page and when a separate call-to-action for Gentinfo is useful for users.
+
+The Gentinfo component should always be used **as is**. The elements, style, layout and content in the Gentinfo component **cannot be changed in anyway**.
```
Comment (hstandaert) [inline styleguide/components/31-molecules/gent-info/README.md]: 😇

```suggestion
The Gentinfo component should always be used **as is**. The elements, style, layout and content in the Gentinfo component **cannot be changed in any way**.
```

File: styleguide/components/31-molecules/gent-info/README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+In the examples the Gentinfo component is shown as it should be included in different languages.
+
+## When not to use this component
+
+* Do not use the Gentinfo component to present other content than the Gentinfo contact information.
+* Do not use the Gentinfo component to present other call-to-actions. For other call-to-actions, use the call-to-action component.
```
Comment (hstandaert) [inline styleguide/components/31-molecules/gent-info/README.md:17]:
```suggestion
* Do not use the Gentinfo component to present other call-to-actions. For other call-to-actions, use the <a href="{{path './cta-block.html'}}">call-to-action</a> component.
```

File: styleguide/components/41-organisms/collection/README.md [hunk 8]
```diff
@@ -1,4 +1,27 @@
-# Teaser collection
+# Collection
+
+A collection is a list of <a href="{{path './teasers.html'}}">teasers</a> used to show teasers of content.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/collection/README.md]:
```suggestion
A collection is a list of <a href="{{path './teaser.html'}}">teasers</a> used to show teasers of content.
```

File: styleguide/components/41-organisms/collection/README.md [hunk 9]
```diff
@@ -1,4 +1,27 @@
...
+
+## How it works
+
+A collection is a <a href="{{path './list.html'}}">list</a> of teaser components that can be shown in a grid of maximum 2, 3, 4 or 5 columns. Depending on the screen resolution, less columns are shown.
+
+On overview and filter pages, <a href="{{path './pagination.html'}}">pagination</a> can be used to divide the content on overview pages with a big amount of content into multiple pages.
+
+On detail pages, an optional "Show more" link can be included, as seen in the example, to link to more related content. The "Show more" always links to another page where more content is shown in an overview, typically an overview or filter page.
+
+Pagination and the "Show more" link can never be used together. Or the teaser collection already shows all content there is and pagination is used to divide the content into multiple pages, or the teaser collection is used to show a selection of related content and a "Show more" link is used to link to all related content.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/collection/README.md]:
```suggestion
Pagination and the "Show more" link can never be used together. Either the teaser collection already shows all content there is and pagination is used to divide the content into multiple pages, or the teaser collection is used to show a selection of related content and a "Show more" link is used to link to all related content.
```

File: styleguide/components/41-organisms/footer/README.md [hunk 10]
```diff
@@ -1,9 +1,47 @@
 # Footer
 
+## When to use this component
+
+All websites and all pages must include the footer.
+
+The footer should be used **as is**. The elements, style, layout and content in the footer **cannot be changed in anyway**, with only a few exceptions when used on separate websites.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/footer/README.md]: I find this kind of vague advice; saying "You can't change it in any way, but you can if you want to make an exception" 🤔

Maybe you could specify in which cases you **are** allowed to change it?

File: styleguide/components/41-organisms/opening-hours-accordion/README.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+## When to use this component
+
+Use the opening hours accordion component to:
+
+* Show an overview of opening hours for a certain location that has one or multiple ways of being open, for instance, free entrance without appointment versus appointment only.
+* Include opening hours but clearly show the type of opening hours and show or hide them depending on if the opening hours are essential for all users or only necessary for a minority of users (progressive disclosure principle).
+* Offer opening hours for this week and for this month and what the opening hours are for each day but also have a quick way to show if the location is open today and what the opening hours are for today.
+
+The opening hours accordion is only to be used on detail pages of locations.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/opening-hours-accordion/README.md]: just to be sure: that implies you can **never** use it on for example a contact page?

File: styleguide/components/41-organisms/programme/README.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+A slot is typically:
+
+* A day or a date
+* A part of a day (for instance, morning, afternoon, evening)
+* A time unit of a few days (for instance, a weekend)
+
+For each slot, the correspondig term there is one description linked with the corresponding term. This description contains the events that take place in that particular slot.
+
+The events that take place in a particular slot are shown using a <a href="{{path './collection.html'}}">collection</a> of <a href="{{path './teasers.html'}}">teasers</a>, shown in a grid of maximum 3 columns. For the teasers, the inverted styling is used.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/programme/README.md]:
```suggestion
The events that take place in a particular slot are shown using a <a href="{{path './collection.html'}}">collection</a> of <a href="{{path './teaser.html'}}">teasers</a>, shown in a grid of maximum 3 columns. For the teasers, the inverted styling is used.
```

File: styleguide/components/41-organisms/programme/README.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+Each teaser represents an event that takes place in a slot and can consist of the following elements:
+
+* Title (required)
+* Date and/or time of the event or subtitle (optional)
+* Image (optional)
+* Summary text (optional)
+* Link (required)
+
+When clicking the link of the teaser, a <a href="{{path './modal.html'}}">fixed-height modal</a> opens containing all information regarding the event:
```
Comment (hstandaert) [inline styleguide/components/41-organisms/programme/README.md]: It is not the fixed-height modal that is used, but the default variant 😉

```suggestion
When clicking the link of the teaser, a <a href="{{path './modal.html'}}">modal</a> opens containing all information regarding the event:
```

File: styleguide/components/41-organisms/theme-footer/README.md [hunk 14]
```diff
@@ -1,19 +1,23 @@
...
-The theme footer can be used if the following conditions apply:
+* The web page is about a **specific City of Ghent theme** or the web page is part of a **separate website**.
+* The City of Ghent theme or separate website has specfic contact information, newsletter subscription and/or social media links specific for that City of Ghent theme or that separate website.
 
-* The web page is part of a specific City of Ghent theme or separate website
-* The City of Ghent theme or separate website has specific (different from the general) contact information *or* newsletters where website users *or* social media channels
+When these conditions apply, the theme footer can be used on all types of pages about the specific City of Ghent theme or on all types of pages on the separate website.
 
-## How to use
+If used, the theme footer should always be placed just above the general footer.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/theme-footer/README.md]:
```suggestion
If used, the theme footer should always be placed just above the <a href="{{path './footer.html'}}">general footer</a>.
```

File: styleguide/components/41-organisms/theme-footer/README.md [hunk 15]
```diff
@@ -1,19 +1,23 @@
...
 
-The theme footer - if used - should be placed just above the general footer.
+## When not to use this component
+
+Do not use the theme footer when the above conditions don't apply.
+
+Do not use the theme footer to present general contact information or general social media links for the City of Ghent. This information is already included in the general <a href="{{path './programme.html'}}">footer</a>.
```
Comment (hstandaert) [inline styleguide/components/41-organisms/theme-footer/README.md]:
```suggestion
Do not use the theme footer to present general contact information or general social media links for the City of Ghent. This information is already included in the general <a href="{{path './footer.html'}}">footer</a>.
```

File: styleguide/components/61-layouts/overview-layout/README.md [hunk 16]
```diff
@@ -0,0 +1,19 @@
+# Overview layout
+
+The overview layout is a one-column layout with one wide column.
+
+## When to use this layout
+
+Use the overview layout to show an overview of content and to link to content in the overview.
+
+The overview page typically uses a <a href="{{path './collection.html'}}">collection</a> of <a href="{{path './teasers.html'}}">teasers</a> to show the overview of the content and link to the detail pages about the content.
```
Comment (hstandaert) [inline styleguide/components/61-layouts/overview-layout/README.md]:
```suggestion
The overview page typically uses a <a href="{{path './collection.html'}}">collection</a> of <a href="{{path './teaser.html'}}">teasers</a> to show the overview of the content and link to the detail pages about the content.
```

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/gent-info/README.md (hunk 7 above)
Comment (mattiasberlamont) [reply styleguide/components/31-molecules/gent-info/README.md:17]: @hstandaert The reason that I did not a link here (yet) is because the Call to action component needs to be refactored (Contact blok and Call to action become one). Because I was not sure what the URL would be, I did not put a link. Will it be this URL? Otherwise, maybe, we could add a TO DO comment?

Comment (hstandaert) [reply styleguide/components/31-molecules/gent-info/README.md:17]: That makes sense! A todo comment is perfect 👍 
```suggestion
<!-- @TODO  Link to the cta component -->
* Do not use the Gentinfo component to present other call-to-actions. For other call-to-actions, use the call-to-action component.
```

File: styleguide/components/41-organisms/footer/README.md (hunk 10 above)
Comment (mattiasberlamont) [reply styleguide/components/41-organisms/footer/README.md]: I made an update to this documentation. Could you check if it is more clear now?

**Review by hstandaert — APPROVED** [review]

File: styleguide/components/41-organisms/opening-hours-accordion/README.md (hunk 11 above)
Comment (delrueba) [reply styleguide/components/41-organisms/opening-hours-accordion/README.md]: That does seem counterproductive.
A lot of effort has been made to make the opening hours api and widget open source and cross platform so it can be used as much as possible and to avoid business adding plain text opening hours on contact pages.

File: styleguide/components/31-molecules/teaser/README.md [hunk 17]
```diff
@@ -1,17 +1,39 @@
...
   screenreaders (WCAG 2.4.3).
-- The teaser overlay-link has visually hidden text content (WCAG 2.4.9).
-- Teaser content must include exactly one link which has the same destination as
+* The teaser overlay-link has visually hidden text content (WCAG 2.4.9).
+* Teaser content must include exactly one link which has the same destination as
   the overlay link (WCAG 2.1.3).
-- If a 'read more' link is used: use aria-label or aria-labelledby to provide
-  screenreader information (WCAG 2.4.4).
\ No newline at end of file
+* If a 'read more' link is used: use aria-label or aria-labelledby to provide
```
Comment (delrueba) [inline styleguide/components/31-molecules/teaser/README.md]: No longer correct, I'll update.

File: styleguide/components/41-organisms/opening-hours-accordion/README.md (hunk 11 above)
Comment (mattiasberlamont) [reply styleguide/components/41-organisms/opening-hours-accordion/README.md]: @hstandaert @delrueba I did not mean that it cannot be used on a contact page (for me, a contact page also has information about a location and opening hours mostly resemble there is some kind of physical location linked to a contact).

I will just update this and describe it as it can be used on (all) detail pages.

**Review by delrueba — APPROVED** [review]

File: styleguide/components/31-molecules/display-switcher/README.md [hunk 18]
```diff
@@ -2,23 +2,35 @@
 
 ## When to use this component
 
-Use the display switcher component to let users switch between display modes of one section of content.
+Use the display switcher component when **a page reload** is triggered to let users:
```
Comment (hstandaert) [inline styleguide/components/31-molecules/display-switcher/README.md:5]: CC @delrueba 

Apologies in advance for bringing this up again... 

I read the comments in #592 about this and would like to point out that this component doesn't have the same functional purpose as the tabs component: 
* **Tabs**: Show most important content on page load and allow the user to gather more information by switching tabs. Each tab clearly contains its own content. 
* **Display switcher**: Show the content in another way and let the user decide which format he prefers. For example grid vs. list

I would state the the user DOES know what to expect, when he clicks "View results on map", which is: View the current page content on a map.

There's plenty of examples of display switchers:
-  [bol.com](https://www.bol.com/nl/l/muziek/N/3132/filter_N/31266+8342+8082+8079/?view=list): no page reload
- [Ebay](https://www.benl.ebay.be/sch/i.html?_from=R40&_sacat=0&_dmd=2&_nkw=nintendo+console&rt=nc&LH_PrefLoc=3): with a page reload

### TLDR;
doesn't matter if a page reload is triggered or not imo..

File: styleguide/components/31-molecules/figure/README.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
+# Figure
+
+## When to use this component
+
+Use the figure component to present an image on a page.
+
+## How it works
+
+The figure component wraps an image in a container and adds a caption for the image. This caption is required.
```
Comment (delrueba) [inline styleguide/components/31-molecules/figure/README.md]:
```suggestion
The figure component wraps an image in a container and adds the benefit of providing an optional caption tag.
```

**Review by delrueba — CHANGES_REQUESTED** [review]

**Review by delrueba — APPROVED** [review]
